### ISSUE TITLE
Can you pease exppose Swift Package Manager option instead of Pod file

### ISSUE THREAD

**Issue by piappstudio** [the-best-is-best/KMMAdmob#1]
Hey, thank you so much for publishing this library. It would be great, if you can expose SMP for the instead instead of Pod file.

**Comment by the-best-is-best** [comment]
Kotlin multi-platform not support it sorry

**Comment by the-best-is-best** [comment]
i will investigate about it thanks

**Comment by the-best-is-best** [comment]
you can use pod packages without make any change in gradle

**Comment by piappstudio** [comment]
True, but my old project doesn't have any POD file in it :-( When I tried to install the pods, it ended up with some conflicts between SPM & Cocopad.

**Comment by the-best-is-best** [comment]
https://github.com/the-best-is-best/KAdmob/ 
use this but this package used in kotlin project but u can see the code of lib and used it in swift code

**Comment by piappstudio** [comment]
Thank you. BTW, I've placed your code as it is in my project, but, I'm short of knowing how to create bridge between swift to kotlin. Could you please help me with bridge configuration steps.

<img width="483" alt="image" src="https://github.com/user-attachments/assets/0e684144-f95a-404e-92b4-e94464748503">

**Comment by the-best-is-best** [comment]
You can't 
 need to update kotlin and compose versions and use pod 

You can't use it without a pod in kotlin 

Issue from jetbrain not supported it yet
